java_src/foam/core/Validateable.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.core;
+
+import java.lang.IllegalStateException;
```

Comment:
You don't need to import anything from the java.lang package since it's already imported by default.